src/profile-logic/profile-data.js

```diff
@@ -914,9 +914,16 @@ export function getTracingMarkers(thread: Thread): TracingMarker[] {
   for (let i = 0; i < markers.length; i++) {
     const data = markers.data[i];
     if (!data) {
-      continue;
-    }
-    if (data.type === 'tracing') {
+      // Add a marker with a zero duration
+      const marker = {
+        start: markers.time[i],
+        dur: 0,
+        name: stringTable.getString(markers.name[i]),
+        title: null,
+        data: null,
+      };
+      tracingMarkers.push(marker);
```

Comment:
This is fine for now, but for follow-up work I feel like a `null` duration rather than 0 would be more helpful for doing the right thing. For instance do a null check for showing the timing information, so we wouldn't show 0ms.

<img width="253" alt="image" src="https://user-images.githubusercontent.com/1588648/29176231-de6c3be4-7db0-11e7-805a-ca66dc63c154.png">